practice/2_python_part_2/task_read_write.py

```diff
@@ -13,3 +13,31 @@
 
     result.txt(content: "23, 78, 3")
 """
+
+from os.path import exists
+
+def read_and_write_to_file():
```

Comment:
It would be better to split reading and writing because it would be easier to test it separately.

Reply:
yes i made those changes but on another branch i will copy and paste code to this one so that it will all make sense